integration/hive/src/main/java/org/apache/carbondata/hive/util/HiveCarbonUtil.java

```diff
@@ -303,8 +321,15 @@ public void commitDropTable(Table table, boolean b) throws MetaException {
     List<String> tokens = new ArrayList();
     StringBuilder stack = new StringBuilder();
     int openingCount = 0;
-    for (int i = 0; i < schema.length(); i++) {
-      if (schema.charAt(i) == '<') {
+    int length = schema.length();
```

Comment:
same comment as above

Reply:
Here I have handled the schema parsing to consider comma-separated values.